src/components/Movies.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import axios from 'axios'
+
+import MovieCard from './MovieCard'
+
+export class Movies extends React.Component {
+  constructor () {
+    super()
+    this.state = {
+      options: [],
+      movie :{},
+      showCard: false
+    }
+  }
+
+  componentDidMount () {
+    this.loadMovies()
+  }
+
+  loadMovies = async () => {
+    const movieList = await axios.get('https://ghibliapi.herokuapp.com/films')
+    const options = movieList.data.map((movie, i) => {
+      return <option key={i} value={movie.id}>{movie.title}</option>
+    })
+    this.setState({ options })
+  }
+
+   loadCard = async(movie) =>{
+    const film = await axios.get(`https://ghibliapi.herokuapp.com/films/${movie}`)
+    this.setState({
+            movie : film.data,
+            showCard: (!movie) ? false : true
```

Comment:
The logic on this line can be done better. I think it can just be:
`showCard: !!movie`
This converts the value into a boolean based on the truthy or falsey valuation